rpm/tayga.spec

```diff
@@ -0,0 +1,60 @@
+Name:		tayga
+Version:	0.9.2
+Release:	1
```

Comment:
Release should be 0 as it is set by the OBS.

Reply:
Seems to be ok now.